### ISSUE TITLE
Project Update - Register Google Maps API Key

### ISSUE THREAD

**Issue by NeoZhangTCL** [mding5692/mapathon#45]
No API key for google maps

**Comment by mding5692** [comment]
Yeah @NeoZhangTCL, I don't think so. Didn't check but that would need to be done, I can register for an API key and attach it back into the code.
